JotunnLib/Entities/SkillConfig.cs

```diff
@@ -0,0 +1,64 @@
+﻿using UnityEngine;
+
+namespace JotunnLib.Entities
+{
+    /// <summary>
+    /// Configuration class for custom skills
+    /// </summary>
+    public class SkillConfig
+    {
+        private string _identifier;
+        public string Identifier
+        {
+            get { return this._identifier; }
+            set
+            {
+                this._identifier = value;
+                if (value.GetStableHashCode() > 1000)
```

Comment:
FYI: at some point GetStableHashCode() was returning a negative number for me, with quite suitable value I assume: 'farefray.valheim.JustAnotherCookingSkill' for identifier. Maybe I'm missing something, but I only managed to make it work with
`Math.Abs(value.GetStableHashCode())`